docs/release-notes/RELEASE-NOTES-0.9.0.md

```diff
@@ -6,45 +6,51 @@ It:
 * Does that TODOFILLMEIN. This impacts the persisted DB;
 * Does that TODOFILLMEIN.
 
-This release introduces backward incompatible changes in the chain format.
-After upgrading your node, you will not have your previous balance (even if you keep your key pair).
+This release introduces backward incompatible changes in the chain format:
+* After upgrading your node, you will not have your previous balance (even if you keep your key pair);
+* Please ensure that you do not reuse a persisted blockchain produced by the previous releases "v0.8.x".
 
 Please join the testnet by following the instructions below, and let us know if you have any problems by [opening a ticket](https://github.com/aeternity/epoch/issues).
 Troubleshooting of common issues is documented [in the wiki](https://github.com/aeternity/epoch/wiki/Troubleshooting).
 
 The instructions below describe:
 * [How to retrieve the released software for running a node](#retrieve-the-software-for-running-a-node);
+* [How to install a node](#install-node);
 * [How to join the testnet](#join-the-testnet).
 
 ## Retrieve the software for running a node
```

Comment:
Also assuming my comments below, I also think all the content below could be reduced to simple list of documentation links.

Reply:
I am open to counter-proposal. Though after considering that I preferred keeping all separate files focused. I.e. docker.md has all Docker related, while the new {configuration,installation,operation}.md files are focused on release binary only.